Detectors/MUON/MCH/Tracking/src/MCHTrackingLinkDef.h

```diff
@@ -0,0 +1,16 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifdef __CLING__
+
+#pragma link C++ class o2::mch::Track + ;
```

Comment:
is this LinkDef file needed in this end ?

Reply:
Useless, indeed.